docs/sources/next/using-k6-browser/recommended-practices/hybrid-approach-to-performance.md

```diff
@@ -0,0 +1,197 @@
+---
+title: 'Hybrid approach to performance'
+heading: 'Hybrid performance with k6 browser'
+head_title: 'Hybrid performance with k6 browser'
+excerpt: 'An example on how to implement a hybrid approach to performance with k6 browser'
+weight: 01
+---
+
+# Hybrid performance with k6 browser
+
+An alternative approach to [browser-based load testing](https://grafana.com/docs/k6/<K6_VERSION>/testing-guides/load-testing-websites/#browser-based-load-testing) that is much less resource-intensive is combining a low amount of virtual users for a browser test with a high amount of virtual users for a protocol-level test. 
```

Comment:
Is `<K6 VERSION>` here correct?

Reply:
Yes, I've looked at the existing examples and seems to be correct :)

Reply:
That's correct! It's how we handle versioned links in the Grafana docs, it'll change based on which version of the docs the user is in. So if they're seeing `latest`, the links on that page will replace `<K6_version>` with `latest`.